components/mosart/docs/tech-guide/index.md

```diff
@@ -1 +1,38 @@
-start of the MOSART Technical Guide
+# MOSART Tech Guide
+
+This Tech Guide includes some technical details about MOSART.
+
+## Table of Conents
```

Comment:
Please don't add an explicit table of contents in the markdown files.  A TOC is generated automatically from the headings and displayed on the right-hand side of the page.

Reply:
Done!